vscode-extension/package.json

```diff
@@ -0,0 +1,94 @@
+{
+	"name": "darklang",
+	"description": "darklang VS Code Extension",
+	"author": "darklang",
+	"version": "0.0.1",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/darklang/dark"
+	},
+	"publisher": "darklang",
+	"categories": [],
+	"keywords": [
+		""
+	],
+	"engines": {
+		"vscode": "^1.74.0"
+	},
+	"main": "./client/out/extension",
+	"contributes": {
+		"languages": [
+			{
+				"id": "darklang",
+				"aliases": [
+					"darklang",
+					"dark"
+				],
+				"extensions": [
+					".dark"
+				],
+				"configuration": "./language-configuration.json",
+        "icon": {
+          "light": "./media/logo-light-transparent.svg",
+          "dark": "./media/logo-light-transparent.svg"
+        }
+			}
+		],
+		"configuration": {
+			"type": "object",
+			"title": "Darklang Extension Config",
+			"properties": {
+				"darklangLsp.maxNumberOfProblems": {
+					"scope": "resource",
+					"type": "number",
+					"default": 100,
+					"description": "Controls the maximum number of problems produced by the server."
+				},
+				"darklangLsp.trace.server": {
+					"TODO": "not sure what this is used for",
+					"scope": "window",
+					"type": "string",
+					"enum": [ "off", "messages", "verbose" ],
+					"default": "off",
+					"description": "Traces the communication between VS Code and the language server."
+				}
+			}
+		},
+    "commands": [{
+      "command": "evalCode",
+      "category": "Darklang",
+      "title": "Evaluate Dark Code"
+    }],
+    "viewsContainers": {
+      "activitybar": [
+        {
+          "id": "canvas-explorer",
+          "title": "Dark Canvas: dark-editor",
+          "icon": "./media/logo-light-transparent.svg"
+        }
+      ]
+    },
```

Comment:
(this is the 'tab' on the left sidenav that our extension can contribute)
unused right now, but will house any custom views that we produce.